python/lsst/daf/butler/script/exportCalibs.py

```diff
@@ -0,0 +1,152 @@
+# This file is part of daf_butler.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from astropy.table import Table
+import logging
+import os
+from .. import Butler, CollectionType
+
+
+log = logging.getLogger(__name__)
+
+
+def parseCalibrationCollection(registry, collection, datasetTypes):
+    """Search a calibration collection for calibration datasets.
+
+    Parameters
+    ----------
+    registry : `lsst.daf.butler.Registry`
+        Butler registry to use.
+    collection : `str`
+        Collection to search.  This should be a CALIBRATION
+        collection.
+
+    Returns
+    -------
+    exportCollections : `list` [`str`]
+        List of collections to save on export.
+    exportDatasets : `list` [`lsst.daf.butler.queries.DatasetQueryResults`]
+        Datasets to save on export.
+    """
+    exportCollections = []
+    exportDatasets = []
+    for calibType in datasetTypes:
+        associations = registry.queryDatasetAssociations(calibType, collections=[collection])
```

Comment:
I think this API takes the standard "collections" types of str, iterable, regex. This means you don't need to turn it into a list and your API here doesn't need to be restrictive. Should this also constrain the search to CALIBRATION collections with `collectionTypes` parameter?